FirebaseInstallations/Source/Library/FIRInstallations.m

```diff
@@ -125,9 +125,7 @@ + (void)validateAppOptions:(FIROptions *)appOptions appName:(NSString *)appName
     [missingFields addObject:@"`FirebaseOptions.googleAppID`"];
   }
 
-  // TODO(#4692): Check for `appOptions.projectID.length < 1` only.
-  // We can use `GCMSenderID` instead of `projectID` temporary.
-  if (appOptions.projectID.length < 1 && appOptions.GCMSenderID.length < 1) {
```

Comment:
just curious: do we still need to check GCMSenderID?

Reply:
Good question. Without `GCMSenderID` IID migration will fail, but FIS will be still functional, so I don't think we should throw an exception in this case. If so, then validation in the method is not needed.

Reply:
+1